packages/graphql/src/translate/queryAST/factory/Operations/UpdateFactory.ts

```diff
@@ -183,6 +186,13 @@ export class UpdateFactory {
                         });
                     } else {
                         for (const op of Object.keys(targetInput[fieldName])) {
+                            const operations = Object.keys(targetInput[fieldName]);
+                            if (operations.length > 1) {
+                                const conflictingOperations = operations.map((op) => `[[${op}]]`);
+                                throw new GraphQLError(
+                                    `Conflicting modification of field ${fieldName}: ${conflictingOperations.join(", ")} on type ${target.name}`
+                                );
+                            }
```

Comment:
looks like this could be moved one line up, before the `for(const op of Object.keys(targetInput[fieldName]))` ?

Reply:
Yeah, good catch, makes more sense outside of the loop